_test_/components/__snapshots__/cards.test.js.snap

```diff
@@ -28,7 +28,7 @@ exports[`<CardComponent /> renders correctly 1`] = `
       className="text-height"
     >
       <p
-        className="cardText card-text"
+        className="card-text card-text"
```

Comment:
These two are exactly the same. Is one meant to override the other?

Reply:
Yes. One comes from how react renders the component to HTML

Reply:
So our own is not overriding the one from react bootstrap?